test/common/gateway.helper.js

```diff
@@ -49,30 +49,23 @@ module.exports.startGatewayInstance = function ({ dirInfo, gatewayConfig }) {
           'lib', 'index.js');
         const gatewayProcess = fork(modulePath, [], {
           cwd: dirInfo.basePath,
-          env: childEnv
+          env: childEnv,
+          stdio: ['pipe', 'pipe', 'pipe', 'ipc']
```

Comment:
do you need `ipc`? As I understand you are just waiting for log output to understand that the process has started (which btw is great idea)

Reply:
It seems like it's mandatory to specify all the parameters, however starting with Node8 you can just put `stdio: 'pipe'` and that'll do the job for you. See [here](https://github.com/nodejs/node/commit/3268863ebc40d1f0beee61b044c492b43fa57fa5#diff-1dea15b19e826b37828eef6c41166ffeR20).

As long we'll support Node 6 — we'll have to supply the whole array.